src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -35,6 +37,17 @@ You can configure the `OtlpExporter` through `OtlpExporterOptions` properties:
 See the [`TestOtlpExporter.cs`](../../examples/Console/TestOtlpExporter.cs) for
 an example of how to use the exporter.
 
+## Environment Variables
+
+The following environment variables can be used to override the default
+values of the `OtlpExporterOptions` (See [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md)).
+
+| Environment variable          | `OtlpExporterOptions` property |
+| ------------------------------| -------------------------------|
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | `Endpoint`                     |
+| `OTEL_EXPORTER_OTLP_HEADERS`  | `Headers`                      |
+| `OTEL_EXPORTER_OTLP_TIMEOUT`  | `TimeoutMilliseconds`          |
```

Comment:
`OTEL_EXPORTER_OTLP_TIMEOUT` has a different unit - seconds - am I correct?

If so I would give a note on that.

Reply:
Actually the specification for this variable does not explicitly say what unit is used.

Reply:
I see. LGTM then